dataprofiler/profilers/data_labeler_column_profile.py

```diff
@@ -27,15 +27,20 @@ def __init__(self, name, data_labeler_dirpath=None, options=None):
             if not isinstance(options, DataLabelerOptions):
                 raise ValueError("DataLabelerColumn parameter 'options' must be"
                                  " of type DataLabelerOptions.")
-            if options.data_labeler_dirpath:
-                data_labeler_dirpath = options.data_labeler_dirpath
```

Comment:
while we remove this check, we now need a new check for the `data_labeler` object